### PR TITLE
Added a new suite

### PR DESCRIPTION
**Pull request by Deepika191**
Added a new suite to run from maven/jenkins

### CODE REVIEW

File: src/test/resources/test-suites/BATS.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+
```
Comment (navinjoy) [inline src/test/resources/test-suites/BATS.xml:3]: Please add more details

File: src/test/resources/test-suites/BATS.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+<suite name="Sanity">
+	<test name="Sanity">
```
Comment (navinjoy) [inline src/test/resources/test-suites/BATS.xml:5]: Use data providers

Comment (Deepika191) [reply src/test/resources/test-suites/BATS.xml:5]: ok

File: src/test/resources/test-suites/BATS.xml (hunk 1 above)
Comment (Deepika191) [reply src/test/resources/test-suites/BATS.xml:3]: will add next commit

**Review by navinjoy — APPROVED** [review]

**Pull request merged**